test/integration/graphql/admin/mutations.spec.ts

```diff
@@ -3,6 +3,8 @@ import { graphql } from "graphql"
 import { User } from "@services/mongoose/schema"
 import { gqlAdminSchema } from "@graphql/admin"
 
+jest.mock("@services/phone-provider", () => require("test/mocks/phone-provider"))
```

Comment:
why do we need mock here?

Reply:
no longer necessary after moving text.ts to a class model